### ISSUE TITLE
about test result

### ISSUE THREAD

**Issue by Latterminer** [dangweili/pedestrian-attribute-recognition-pytorch#1]
Thanks for your excellent work and code!
But i get  a issue when testing the model trained by myself using peta dataset, the result is far different from the labels.
Do you have the pretrained model work well that i can do some test?
Thanks for your reply.

**Comment by dangweili** [comment]
OK. You can try this one:
链接: https://pan.baidu.com/s/1JfxGDe0Wj2PG-WVHiB394Q 提取码: n4gi
The results should be:
mA: 0.8423 
Acc: 0.7917, Prec: 0.8796, Rec: 0.8559, F1: 0.8676

**Comment by trikim** [comment]
@dangweili , thanks for providing the pretrained model.
I have tested the model by 'sh script/experiment/test.sh' and also get poor performance. I'm doubt the weights are not loaded to the model correctly. 
what's more, even though the model_weight_file is not exist, the test.sh can run.

**Comment by trikim** [comment]
@dangweili Thank you very much! The problem is solved.
I have trained the model on peta dataset, but get a little poorer performance.
The results are:
mA: 0.8367
Acc: 0.7873, Prec: 0.8771, Rec: 0.8503, F1: 0.8635
Have you done some other operations(such as data augmentation) except the command "sh ./script/experiment/train.sh" ?

**Comment by srika91** [comment]
@dangweili  I am not able to download the model from the baidu link.Can u upload it in google cloud?

**Comment by dangweili** [comment]
@srika91 Please try this one:
https://drive.google.com/file/d/1_1JeQ3jkuKwgH7NUMai1aMZYh-s0rDie/view?usp=sharing

**Comment by nakill** [comment]
hi,
why Resnet50 (Acc: 0.787 ) <  Alexnet (Acc: 82.6 in paper). As far as i know Resnet50 better Alexnet.
Thks.

**Comment by Addie11** [comment]
Can anyone please share the  RAP dataset.

**Comment by daiwc** [comment]
> Can anyone please share the RAP dataset.

if you got rap dataset, please share with me ,thanks a lot!!

**Comment by BinchaoPeng** [comment]
> @dangweili Thank you very much! The problem is solved.
> I have trained the model on peta dataset, but get a little poorer performance.
> The results are:
> mA: 0.8367
> Acc: 0.7873, Prec: 0.8771, Rec: 0.8503, F1: 0.8635
> Have you done some other operations(such as data augmentation) except the command "sh ./script/experiment/train.sh" ?

do you have the demo about the model，how to use it
